src/plugins/spark/features/TempSensorOneWire/TempSensorOneWireWidget.vue

```diff
@@ -38,9 +38,12 @@ export default class TempSensorOneWireWidget extends BlockWidget {
         <q-item-section>This sensor is invalid</q-item-section>
       </q-item>
       <q-item v-else dark>
-        <q-item-section>Value</q-item-section>
         <q-item-section>
-          <big>{{ block.data.value | unit }}</big>
+          <q-item-label caption>Value</q-item-label>
```

Comment:
This pattern (q-item-section / caption / value / smaller notation) can probably be extracted in a separate component.